execution_engine/src/core/runtime/mod.rs

```diff
@@ -2994,3 +3000,22 @@ where
         Ok(Ok(()))
     }
 }
+
+#[cfg(feature = "test-support")]
+fn dump_debug_info(error: &wasmi::Error, instance: wasmi::ModuleRef) {
+    // Currently, we're only interested in the details for the `Unreachable` error.
+
+    if let wasmi::Error::Trap(trap) = error {
+        if let TrapKind::Unreachable = trap.kind() {
+            eprintln!("Wasm execution error: TrapCode::Unreachable");
```

Comment:
I don't think we need this personally, but if you prefer to keep it, maybe it should say TrapKind rather than TrapCode?

Reply:
Removed in https://github.com/casper-network/casper-node/commit/7f8dcebbde140e9be8b10500c7fb9ac5ff355fe7